ci/testsettings.py

```diff
@@ -30,7 +30,7 @@
             'COLLECTION': 'parasolr_test',
             # aggressive commitWithin for test only
             'COMMITWITHIN': 750,
-            'CONFIGSET': 'basic_configs'
+            'CONFIGSET': '_default'
```

Comment:
For the build matrix with my proposed approach, we would need to leave this off but figure out how to specify solr version based on the environment variable.